transformation/src/main/scala/org/broadinstitute/monster/hca/HcaPipelineBuilder.scala

```diff
@@ -392,21 +384,10 @@ object HcaPipelineBuilder extends PipelineBuilder[Args] {
         }
 
       // Generate file ingest requests from descriptors. Deduplicate by the content hash.
-      val keyedIngestRequests = validatedDescriptors.map {
+      val fileIngestRequests = validatedDescriptors.map {
         case (_, descriptor) =>
           generateFileIngestRequest(descriptor, inputPrefix)
-      }.distinctByKey.values.keyBy(request => request.read[String]("source_path"))
-
-      val fileIngestRequests =
-        keyedIngestRequests.leftOuterJoin(files).flatMap {
-          case (filename, (_, None)) =>
```

Comment:
Is this the extra validation that you mentioned "might cause issues in the future"?

Reply:
yeah, because there is a future where we want to update metadata about files that are already in the repo